src/bluesky/run_engine.py

```diff
@@ -2353,7 +2353,7 @@ def cancel_status_task_if_error(fut: asyncio.Future[list[asyncio.Future]]):
                     if done:
                         self._call_waiting_hook(None)
                         self._seen_wait_and_move_on_keys.remove(group)
-            return done
+        return done
```

Comment:
If there are no futures, are we done?

```suggestion
        else:
            done = True
        return done
```